pkg/test/framework/components/echo/kube/testdata/multiversion.yaml

```diff
@@ -47,7 +47,7 @@ spec:
         args:
           - --port
           - "8090"
-          - --port
+          - --tcp
```

Comment:
It's better to just add a new tcp port instead of changing the current one.

Reply:
Port 9000 is tcp port. This is the testdata for comparison.